src/utils/db.ts

```diff
@@ -346,9 +349,15 @@ class DB {
     })
   }
 
-  async createGroup(groupId, groupName, user1Email) {
+  async createGroup(groupId: String, groupName: String, user1Email: String) {
     const session = this.localDriver.session({ database: "neo4j" });
 
+    // Currently initializes channels when a group is created
+    // TODO: Replace this with code to add channels through the add channel button
+    this.createChannel(groupId.toString() + "general", groupId)
```

Comment:
we can actually keep general channel here int future, so all groups start off with a general channel